cardano-node/src/Cardano/Node/Tracing/Tracers/ChainDB.hs

```diff
@@ -2090,10 +2090,42 @@ instance MetaTrace V1.BackingStoreValueHandleTrace where
     , Namespace [] ["Statted"]
     ]
 
-
 instance LogFormatting V2.FlavorImplSpecificTrace where
+  forMachine _dtal V2.FlavorImplSpecificTraceInMemory =
+    mconcat [ "kind" .= String "InMemory" ]
+  forMachine _dtal V2.FlavorImplSpecificTraceOnDisk =
+    mconcat [ "kind" .= String "OnDisk" ]
+
+  forHuman V2.FlavorImplSpecificTraceInMemory =
+    "An in-memory backing store event was traced"
+  forHuman V2.FlavorImplSpecificTraceOnDisk =
+    "An on-disk backing store event was traced"
 
 instance MetaTrace V2.FlavorImplSpecificTrace where
+  namespaceFor V2.FlavorImplSpecificTraceInMemory =
+    Namespace [] ["InMemory"]
+  namespaceFor V2.FlavorImplSpecificTraceOnDisk =
+    Namespace [] ["OnDisk"]
+
+  severityFor (Namespace _ ["InMemory"]) _ = Just Info
+  severityFor (Namespace _ ["OnDisk"])   _ = Just Info
+  severityFor _                          _ = Nothing
+
+  -- suspicious
```

Comment:
You could have omitted this one. It defaults to public

Reply:
I wanted to add explicit cases (and the "suspicious" comment) because I thought it'd be the kind of thing that's easy to forget about if / when other constructors get added -- I don't think it makes a big difference though, so I don't mind just using the default 🤷